docs/INSTALLATION.md

```diff
@@ -32,29 +34,23 @@ Follow the steps below to set up the repo for development
     git clone https://github.com/avantifellows/plio-frontend.git
     cd plio-frontend
     ```
-2. Install the dependencies
-    #### Development
+2. Make sure Docker is installed and docker version is running fine.
```

Comment:
same comment from the backend repo about a note on starting Docker Desktop when doing this locally.